src/app/client/src/app/app.component.ts

```diff
@@ -281,7 +283,25 @@ export class AppComponent implements OnInit, OnDestroy {
       this.checkFrameworkSelected();
     }
   }
-
+  public getOrgDetails() {
+    let slug = this.userService.slug;
+    return this.orgDetailsService.getOrgDetails(slug).pipe(
+      tap(data => {
+        this.cacheService.set('orgDetailsFromSlug', data, {
+          maxAge: this.browserCacheTtlService.browserCacheTtl
+        });
+      })
+    );
+  }
+  public checkForCustodianUser() {
+    this.orgDetailsService.getCustodianOrgDetails().subscribe((custodianOrg) => {
+      if (_.get(this.userService, 'userProfile.rootOrg.rootOrgId') === _.get(custodianOrg, 'result.response.value')) {
+        this.userService.setIsCustodianUser(true)
```

Comment:
Where is the logic after landing to igot user is login user Gmail. Then where are we setting 
this.userService.setIsCustodianUser(true)